src/js/angular/import/templates/settingsModal.html

```diff
@@ -5,264 +5,301 @@ <h4 class="modal-title">{{'import.settings' | translate}}</h4>
 <div class="modal-body">
     <form name="settingsForm" id="settingsForm" class="settings-form">
         <div class="form-horizontal">
-        <div class="form-group row">
```

Comment:
Fix the formatting please. Check if your IDE isn't configured to format the whole file. Revert the file and reapply the changes without formatting the entire file.

Reply:
I am not sure if I successfully reverted the file format but the IDE shows me that this is the previous format with my change.

Reply:
Fixed file alignment.